sales/sale_amazon/apply.rst

```diff
@@ -0,0 +1,148 @@
+================================
+Apply for Amazon MWS Access Keys
+================================
+
+.. _amazon/developer-form:
+
+Submit the Amazon MWS Developer Registration and Assessment Form
+================================================================
+
+In order to synchronize your Amazon orders with Odoo, Amazon MWS access keys are required.
+They can be obtained by submitting the **Amazon MWS Developer Registration and Assessment form** to
+register as a developer. Once recognized by Amazon as a developer (i.e. you make use of an
+application connecting to MWS), you will be granted Amazon MWS access keys.
+
+First, visit the `Amazon Marketplace Web Service documentation
+<http://docs.developer.amazonservices.com/en_US/dev_guide/DG_Registering.html>`_ and follow the
+instructions to register as a developer. Take care to choose the form "I represent a seller
+organization integrating with Amazon MWS for its own selling account only.".
+
+Fill out the *Developer Registration and Assessment form* as suggested below and provide your own
+contact information in the **Developer contact information** section. In the **Business use
+information** section, select the correct region of your seller account. For the other sections,
+adapt your responses in accordance with your business case.
+
+Give a particular attention to the **Merchant Fulfilled Shipping** function. It should only be
+checked if you ship your products yourself. You should uncheck it if you sell exclusively with the
+*Fulfillment by Amazon* service. Please note that requesting this function is not recommended if you
+are hosted on *Odoo Online* or *Odoo.sh* as the additional security requirements asked by Amazon may
+not be met by Odoo.
+
+.. warning::
+   Depending on several factors (your region, whether you checked the **Merchant Fulfilled
+   Shipping** function (i.e. you request access to Personally Identifiable Information of your
+   customers), etc.), Amazon may request you to fill out a second form before granting you MWS
+   access keys. As that form depends on the data protection policy of the region of your seller
+   account (e.g. GDPR in Europe), we cannot provide you with a pre-filled form. Instead, the answers
+   of questions related to Odoo are listed in the `Answer the Additional Form`_ section.
+
+.. tip::
+   If you need assistance for your application for Amazon MWS access keys, `submit a support ticket
+   to Odoo <https://www.odoo.com/help>`_.
+
+.. image:: ./media/dev_form.png
+
+Answer the Additional Form
+==========================
+
+This section lists all questions asked by Amazon in additional forms. The answers are tailored for
+*Odoo Online* and *Odoo.sh*. If you did not receive any additional form after your :ref:`initial
+application for Amazon MWS keys <amazon/developer-form>`, disregard the following.
+
+
+.. warning::
+   If you host Odoo yourself (*on-premise*), adapt the answers related to hosting to your own
```

Comment:
If you are not hosted on Odoo.com or Odoo.sh (*on-premise*), you should adapt your answers for the Hosting section according to your own infrastructure and data protection policy.

=> they may not be hosting *themselves*, as they might have a partner